ui-service/angular.json

```diff
@@ -133,5 +133,8 @@
       }
     }
   },
-  "defaultProject": "cg-ui"
+  "defaultProject": "cg-ui",
+  "cli": {
+    "analytics": "bf15d83f-c6cd-4314-8950-9d3442aba6dc"
```

Comment:
What is this `analytics` line?

Reply:
It is for gathering some data from the Angular team and use to improve CLI(). I think because this is an opensource application, it would be good if we help with that. 
[https://angular.io/cli/usage-analytics-gathering](URL)
and this is the data they are gathering 
[https://angular.io/analytics](URL)

If this is not OK with us, I will disable usage analytics

Reply:
The issue is that I do not understand what type of analytics this is. Depending on that, we either have to add a banner to inform the user or we even have to ask the user whether we are allowed to use it.

So from my point of view, having an analysis is in general nice. But it may add a lot of work that we don't want to have at the moment. There are enough other issues to solve :wink:

Reply:
you right, I didn't attend to this aspect. I will remove this and update my pull request.